data-load/station_imtd_threshold/get-imtd-api-threshold.js

```diff
@@ -0,0 +1,95 @@
+const axios = require('axios')
+const { parseThresholds } = require('./parse-thresholds')
+
+// Connect to database and generate list of station ids
+
+const { Pool } = require('pg')
+
+const connectionString = process.env.FLOOD_SERVICE_CONNECTION_STRING
+
+const getDBStuff = async () => {
+  const pool = new Pool({ connectionString })
+  const result = await pool.query('select distinct rloi_id from rivers_mview where rloi_id is not null order by rloi_id asc')
+  console.log('Number of stations from database: ', result.rows.length)
+  await pool.end()
+  return result.rows
+}
+
+// End database stuff
+
+// Create csv header
+
+const fs = require('fs')
+
+console.log('Writing CSV header')
+const csvHeader = 'station_id,fwis_code,fwis_type,direction,value\n'
+fs.writeFile('./station-threshold.csv', csvHeader, err => {
+  if (err) {
+    console.log(err)
+  }
+})
+
+const getData = (stationId) => {
+  return new Promise((resolve, reject) => {
+    axios
+      .get(`https://imfs-prd1-thresholds-api.azurewebsites.net/Location/${stationId}?version=2`)
+      .then(res => {
+        try {
+          const thresholds = parseThresholds(res.data[0].TimeSeriesMetaData)
```

Comment:
I suspect we don't need multiple thresholds for a given stationId, area and direction inserted into the DB which are then reduced by the SQL min function. 

e.g. (from station-threshold.csv)

```
1071,064WAF324,A,u,39
1071,064WAF324,A,u,39.5
```

We could do the calculation of Min here and not end up with duplicates.

Reply:
Discussed with Product Owner and going to retain the flexibility afforded by storing all the records for the time being.